addons/stock/static/src/views/picking_form/stock_move_one2many.js

```diff
@@ -49,8 +75,8 @@ export class StockMoveX2ManyField extends X2ManyField {
         if (this.canOpenRecord && !record.isNew) {
             const dirty = await record.isDirty();
             if (dirty && 'quantity' in record._changes) {
-                await record.model.root.save({ reload: true });
-                record = record.model.root.data[this.props.name].records.find(e => e.resId === record.resId);
+                await record._parentRecord.save({ reload: true });
```

Comment:
```suggestion
                await record.model.root.save({ reload: true });
```
I think it's still valid to save the root model. Next line is okay though.

Reply:
Hello @clesgow,

If I apply 'model.root' instead of '_parentRecord', there is no synchronization between stock move to stock move line in batch transfers.

Thank you.

Reply:
Indeed, good catch! :+1:

Note: If you set a quantity in the move then open the details to close it immediately without doing anything, even if you hit `Discard`, the change in quantity is still saved regardless (due to the `save()` above).
Not for this PR though, as it may be overly complicated and it's not good practice, since you should be using the `Detailed Operations` tab in the first place for that anyway.